hierarchy/store.go

```diff
@@ -129,6 +140,32 @@ func cloneNodes(connection bolt.Conn, instanceID, codeListID, dimensionName stri
 	return err
 }
 
+func countNodes(conn bolt.Conn, id string) (count int64, err error) {
+	query := fmt.Sprintf("MATCH (n:`%s`) RETURN COUNT(n);", id)
+
+	rowCursor, err := conn.QueryNeo(query, nil)
+	if err != nil {
+		return
+	}
+	defer func() {
+		if err := rowCursor.Close(); err != nil {
+			log.ErrorC("Deferred rowCursor.Close", err, nil)
+			panic(err)
```

Comment:
Not sure that we would want to panic here

Reply:
good spot